Doc/c-api/apiabiversion.rst

```diff
@@ -58,5 +58,7 @@ See :ref:`stable` for a discussion of API and ABI stability across versions.
    Thus ``3.4.1a2`` is hexversion ``0x030401a2`` and ``3.10.0`` is
    hexversion ``0x030a00f0``.
 
+   Starting with Python 3.11, this information is also available via the
```

Comment:
Please document also the variable using the `.. c:var:: PY_VERSION_HEX` markup and add a `.. versionadded:: 3.11` markup.